.env.example

```diff
@@ -47,4 +47,13 @@ MINIO_ROOT_USER=EXAMPLEID
 CLAMAV_ENDPOINT=https://clamav-rest:9443
 # if running locally
 # CLAMAV_ENDPOINT=http://localhost:8081
-REDIS_PASS=SUPERSECUREPASSWORD
\ No newline at end of file
+REDIS_PASS=SUPERSECUREPASSWORD
+
+#SMTP Settings
+SMTPHOST=smtp.test.gov
+SMTPUSER=user@test.gov
+SMTPPASS=supersecurepassword
+SMTPPORT=465
+# SMTPSECURE should be set to true if SMTPPORT is 465. Otherwise it should be false
+SMTPSECURE=true
+FROMEMAILADDRESS=test@test.gov
```

Comment:
Should be make these with camel case for a bit easier read? `FROM_EMAIL_ADDRESS`/`SMPT_SECURE`?

Reply:
I think that makes sense